docs/src/language_servers.md

```diff
@@ -793,7 +807,7 @@ See [Javascript/TypeScript](#javascripttypescript).
     <!-- how to call: see https://github.com/nvarner/typst-lsp/blob/master/editors/vscode/src/extension.ts -->
     ```jsonc title="Packages/User/Default.sublime-commands"
     [
-        // ...
+        // typst-lsp:
```

Comment:
`// ...` were supposed to signify that there might be other content in this file also.

Reply:
fixed